plugins/package-managers/bower/src/main/kotlin/BowerDependencyHandler.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.bower
+
+import java.io.File
+
+import org.ossreviewtoolkit.analyzer.PackageManager.Companion.processProjectVcs
+import org.ossreviewtoolkit.analyzer.parseAuthorString
+import org.ossreviewtoolkit.downloader.VersionControlSystem
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.Issue
+import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.PackageLinkage
+import org.ossreviewtoolkit.model.Project
+import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.model.utils.DependencyHandler
+
+internal class BowerDependencyHandler : DependencyHandler<PackageInfo> {
+    override fun identifierFor(dependency: PackageInfo) = dependency.toIdentifier()
+
+    override fun dependenciesFor(dependency: PackageInfo) = dependency.dependencies.values.toList()
+
+    override fun linkageFor(dependency: PackageInfo) = PackageLinkage.DYNAMIC
+
+    override fun createPackage(dependency: PackageInfo, issues: MutableCollection<Issue>) = dependency.toPackage()
+}
+
+internal fun PackageInfo.toIdentifier() =
```

Comment:
Do all of the below need to be `internal` visible?

Reply:
I've made `toVcsInfo()` and `toPackage()` private now because I could, although it feels a bit inconsistent WRT `toProject()` which needs to be internal.